plugins/userdata/JetUserData.cc

```diff
@@ -167,14 +173,23 @@ void JetUserData::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
 
 	edm::ESHandle < JetCorrectorParametersCollection > jetCorrectorCollection;
 	iSetup.get<JetCorrectionsRecord>().get(jecUncertainty_, jetCorrectorCollection);
+
 	JetCorrectorParameters const & jetCorrectorParams = (*jetCorrectorCollection)["Uncertainty"];
 	JetCorrectionUncertainty jecUnc(jetCorrectorParams);
 	const JetCorrector* jetCorrector(JetCorrector::getJetCorrector(jetCorrectionService_, iSetup));
 
-//	bool isSimulation = !iEvent.isRealData();
-
-	// https://twiki.cern.ch/twiki/bin/view/CMS/JetEnergyScale
-	// https://twiki.cern.ch/twiki/bin/view/CMS/JECAnalysesRecommendations
+	JME::JetResolution resolution;
+	JME::JetResolutionScaleFactor resolution_sf;
+	if (!iEvent.isRealData()) {
+		if( jerFromFile_ == true ){
+			resolution = JME::JetResolution(resFile_);
```

Comment:
Please do. But since we will need it in the future, could you please create an issue about it so it is not forgotten?